Client/src/Components/DataCell/DataCell.js

```diff
@@ -64,6 +66,11 @@ export default function DataCell() {
         const i = filename.lastIndexOf('.')
         return (i < 0) ? '' : filename.substr(i)
     }
+    //to do: refactor into proper txt file

```

Comment:
remove messages like this into the assets folder, you could call it CellViewMessages.tsx, not a txt tho since you want the constants right, or it could also be a json file with messages

Reply:
i removed/relocated them into assets folder and managed to import it. thanks for the feedback

Reply:
remove the todo comment :)

Reply:
i thought i did?